src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml.cpp

```diff
@@ -125,6 +125,15 @@ void EquationInputArea::EquationTextBox_Submitted(Object ^ sender, MathRichEditB
         || (submission->Source == EquationSubmissionSource::FOCUS_LOST && submission->HasTextChanged && eq->Expression != nullptr
             && eq->Expression->Length() > 0))
     {
+        auto eqButton = VisualTree::FindDescendantByName(tb, "EquationButton");
```

Comment:
The IsLineEnabled property should be bound to a property on the EquationTextBox that has a TemplateBinding to the EquationButton->IsChecked property. Using the VisualTree is a more likely to break silently in the future as people change the XAML and eqButton suddenly becomes a nullptr with no compile error.